.github/workflows/golangci-lint.yml

```diff
@@ -17,4 +17,4 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.34
-          args: -v
+          args: -v --timeout 2m
```

Comment:
You may want to bump this higher. We have 5m for kapp-controller. The golangci-lint action can be flakey some times, so probably best to be conservative.

Reply:
Sure. Thanks